src/dms/core/EdFi.DataManagementService.Core/OpenApi/OpenApiDocument.cs

```diff
@@ -221,28 +221,144 @@ private void InsertNewTags(JsonArray newTagObjects, JsonNode openApiCoreResource
     }
 
     /// <summary>
-    /// Finds the openApiExtensionDescriptors and openApiExtensionResources in extension ApiSchemaDocument.
+    /// Merges OpenAPI fragments from source into target
     /// </summary>
-    private List<JsonNode> FindOpenApiExtensionFragments(JsonNode extensionApiSchemaRootNode, string section)
+    private static void MergeOpenApiFragments(JsonNode source, JsonNode target)
```

Comment:
New code that combines the fragments from each resource. Follow-on domain filtering will happen here